### ISSUE TITLE
Is there a way to remove the unminified react-native javascript in the .ipa file?

### ISSUE THREAD

**Issue by jduffy7** [fastlane/fastlane#15563]
### Question Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Question Subject
Why is there both a minified and unminified version of my javascript in my .ipa file? And further, can I remove the unminified version of the javascript?

For security purposes, I don't want someone to be able to download my .ipa file from the appstore and see all of my javascript source code (I believe they can extract the .ipa with a jailbroken phone).

#### Question Description
I am successfully generating builds and uploading them to TestFlight. However, when I run a local build (Fastfile seen below) to try and inspect the files that I'm uploading to TestFlight, I get an .ipa file that I open and can see all of my unminified javascript in a /src/ folder (blue box) in addition to minified javascript (red box):

![image](https://user-images.githubusercontent.com/40577725/67807095-88328380-fa6a-11e9-9413-f5f4c332646e.png)

Is there a way to remove this /src/ folder from the .ipa output?

Fastfile command that generates the output .ipa file:
```
platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :local do
    match(type: "adhoc")
    build_app(workspace: "mobile.xcworkspace", scheme: "mobile", export_xcargs: "-allowProvisioningUpdates", export_method: "ad-hoc")
  end
end
```

#### fastlane env
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15                                                          |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.21.0 (Apple Git-122)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15 (19A602)                                        |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.0                                                           |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
    increment_build_number(xcodeproj: "mobile.xcodeproj")
    match(type: "appstore")
    build_app(workspace: "mobile.xcworkspace", scheme: "mobile", export_xcargs: "-allowProvisioningUpdates")
    upload_to_testflight
  end
  
  lane :local do
    match(type: "adhoc")
    build_app(workspace: "mobile.xcworkspace", scheme: "mobile", export_xcargs: "-allowProvisioningUpdates", export_method: "ad-hoc")
  end
  
  lane :release do
    increment_build_number(xcodeproj: "mobile.xcodeproj")
    match(type: "appstore")
    build_app(workspace: "mobile.xcworkspace", scheme: "mobile", export_xcargs: "-allowProvisioningUpdates")
    upload_to_app_store
    
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("REDACTED") # The bundle identifier of your app

itc_team_id("REDACTED") # App Store Connect Team ID
team_id("REDACTED") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.134.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| CFPropertyList            | 2.3.6        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.13.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.6.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.0        |
| tty-cursor                | 0.7.0        |
| tty-spinner               | 0.9.1        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.67.0       |
| faraday                   | 0.17.0       |
| unf_ext                   | 0.0.7.6      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.13.1       |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| json                      | 2.1.0        |
| mini_magick               | 4.9.5        |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.5        |
| bundler                   | 1.17.2       |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.6        |
| jwt                       | 2.1.0        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mime-types-data           | 3.2019.1009  |
| mime-types                | 3.3          |
| multi_json                | 1.14.1       |
| signet                    | 0.12.0       |
| memoist                   | 0.16.0       |
| os                        | 1.0.1        |
| googleauth                | 0.6.7        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.23.9       |
| google-cloud-env          | 1.2.1        |
| google-cloud-core         | 1.3.2        |
| digest-crc                | 0.4.1        |
| google-cloud-storage      | 1.16.0       |
| emoji_regex               | 1.0.1        |
| forwardable               | 1.2.0        |
| logger                    | 1.3.0        |
| stringio                  | 0.0.2        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.1.0        |
| strscan                   | 1.0.0        |
| date                      | 2.0.0        |
| fileutils                 | 1.1.0        |
| etc                       | 1.0.1        |
| io-console                | 0.4.7        |
| zlib                      | 1.0.0        |
| libxml-ruby               | 3.1.0        |
| rexml                     | 3.1.9        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
</details>

*generated on:* **2019-10-29**
</details>

**Comment by janpio** [comment]
That is not really a fastlane question - it just uses Xcode to build your app, so the result so should be identical to when you build with Xcode itself.

**Comment by jduffy7** [comment]
I see, thank you. I was curious if there were any build options that fastlane accepts that would achieve the minification I'm looking for. I couldn't find anything in the docs. I'm guessing the answer is no, but I don't know if xcode has any kind of option like that either.

**Comment by janpio** [comment]
If Xcode has one, fastlane also supports this.

**Comment by jduffy7** [comment]
It turned out that we had a configuration in our build that was explicitly moving the `/src/` folder into the build via Xcode's Build Phases > Copy Bundle Resources. So I removed that configuration. Note for anyone in the future that when I did that, our images were no longer able to be resolved, so I had to add `--assets-dest='./ios'` to the `bundle` command that creates our  `main.jsbundle`, and then add that folder to Copy Bundle Resources.

This can be closed.
